### ISSUE TITLE
Gauge charts issue

### ISSUE THREAD

**Issue by 41south** [poblabs/weewx-belchertown#496]
Weewx 4.0.0 and BT 1.2 (release version)

I tried to add a rain gauge using the gauge style as per the Wiki example

[Wiki Guages](https://github.com/poblabs/weewx-belchertown/wiki/Belchertown-Charts-Documentation#gauge-chart)

[[exampleGauge]]
        title = Example gauge chart
        type = gauge
        [[[outTemp]]]
            rounded = 1
            color = "#f7a35c"

Unfortunately all this gives me is a speedometer type of gauge (complete black circle with the 'odometer' type number in the middle) with a black line at the 3o'clock - nothing like what is depicted in the wiki. 
Is there a problem with the wiki example code or have I missed something out?

Thanks

**Comment by uajqq** [comment]
It sounds like you're not on the most recent version of the skin. Try to download the latest version from Github, reinstall, restart weewx, and try then.

**Comment by 41south** [comment]
Ah so is this only supported on the latest beta release perhaps? I'm on
production release 1.2

Will give that a go, thanks

**Comment by c0by85** [comment]
Got the same issue, can't fix it.

```
[exampleGauge]
[[exampleGauge]]
title = Example gauge chart
type = gauge
[[[outTemp]]]
```

![image](https://user-images.githubusercontent.com/25707747/110928453-2e60ef80-8327-11eb-8d1d-f47d6f9cd3bb.png)

**Comment by uajqq** [comment]
Not sure why that wouldn't be working. Are you on the latest version of the skin?

**Comment by c0by85** [comment]
i think so.

`Belchertown Skin Version: 1.2`

**Comment by uajqq** [comment]
Try to re-download the latest version from GitHub and reinstall. There have been a few updates in the last few months. Is that the only gauge on your site, or are some others working?

**Comment by c0by85** [comment]
this was only an test but it doesn't work, so there will be no gauge at the moment. when i add `time_length = 1` i will see an speedometer needle.

**Comment by uajqq** [comment]
Have you tried downloading the latest skin and reinstalling it? I think there might be some outdated files in your setup.

**Comment by 41south** [comment]
I fixed this downloading the latest beta of 1.3b1 so closing
